array-methods/filter.js

```diff
@@ -0,0 +1,23 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evenNumbers = numbers.filter(function (element) {
+  return element % 2 === 0;
```

Comment:
You use an arrow function below, but not here? Once you get used to arrow functions, you'll use them practically everywhere!

Reply:
Thank you, will do !